playbook-website/app/javascript/components/Website/src/layouts/Sidebar/MenuData/GuidesNavItems.ts

```diff
@@ -0,0 +1,18 @@
+export const GuidesNavItems = [
+  {
+    name: "HTML & CSS Setup",
+    link: "/guides/getting_started/HTML_&_CSS",
+  },
+  {
+    name: "Ruby & React Setup",
+    link: "/guides/getting_started/rails_&_react_setup",
+  },
+  {
+    name: "React Setup",
+    link: "/guides/getting_started/react_setup",
+  },
+  {
+    name: "Ruby on Rails Setup",
+    link: "/guides/getting_started/ruby_on_rails_setup",
+  },
+];
```

Comment:
I attempted to use the data that you had created. However, the controller that this data is in, the data only becomes available on visiting of the actual url (since it's a different one that my current view). In other words, my JSON was not being populated with the data, until i actual went to those pages, and then returned back to my beta/kits.json (then, i would see the data).

My plan was to separate the data into a rails concern that could be shareable across multiple controllers. And then bring that item into my controller as an include at the top. However, I was going to do this in a follow-up story because it ended up taking a good bit of time.